_includes/config-guide/custom-logger-handler-examples.md

```diff
@@ -0,0 +1,182 @@
+        * Logging level
+        *
+        * @var int
+        */
+       protected $loggerType = MonologLogger::ERROR;
+
+       /**
+        * File name
+        *
+        * @var string
+        */
+       protected $fileName = '/var/log/my_custom_logger/error.log';
+   }
+   ```
+
+1. Define it for the specific handler type in the `di.xml`: 
+
+   For example, `Vendor\ModuleName\Logger\Handler\ErrorHandler` will be defined for the error handler:
+
+   ```xml
+   <virtualType name="MyCustomLogger" type="Magento\Framework\Logger\Monolog">
+       <arguments>
+           <argument name="handlers" xsi:type="array">
+               <item name="error" xsi:type="object">Vendor\ModuleName\Logger\Handler\ErrorHandler</item>
+           </argument>
+       </arguments>
+   </virtualType>
+   ```
+
+   where `MyCustomLogger` is the unique identifier of [virtual type]({{page.baseurl}}/extension-dev-guide/build/di-xml-file.html#virtual-types).
+
+1. Specify a class name that you will use in your custom logger handler.
+
+   For example, you will use a custom logger handler in the `Vendor\ModuleName\Observer\MyObserver` class:
+
+   ```xml
+   <type name="Vendor\ModuleName\Observer\MyObserver">
+       <arguments>
+           <argument name="logger" xsi:type="object">MyCustomLogger</argument>
+       </arguments>
+   </type>
+   ```
+
+   Source code of `Vendor\ModuleName\Observer\MyObserver` class:
+
+   ```php
+   <?php
+   /**
+    * @author Vendor
+    * @copyright Copyright (c) 2019 Vendor (https://www.vendor.com/)
+    */
+   declare(strict_types=1);
+
+   namespace Vendor\ModuleName\Observer;
+
+   use Psr\Log\LoggerInterface as PsrLoggerInterface;
+   use Exception;
+   use Magento\Framework\Event\ObserverInterface;
+   use Magento\Framework\Event\Observer;
+
+   /**
+    * Class MyObserver
+    */
+   class MyObserver implements ObserverInterface
+   {
+       /**
+        * @var PsrLoggerInterface
+        */
+       private $logger;
+
+       /**
+        * MyObserver constructor.
+        *
+        * @param PsrLoggerInterface $logger
+        */
+       public function __construct(
+           PsrLoggerInterface $logger
+       ) {
+           $this->logger = $logger;
+       }
+
+       /**
+        * @param Observer $observer
+        */
+       public function execute(Observer $observer)
+       {
+           try {
+               // some code goes here
+           } catch (Exception $e) {
+               $this->logger->error($e->getMessage());
+           }
+       }
+   }
+   ```
+
+1. The class `Vendor\ModuleName\Logger\Handler\ErrorHandler` will be injected into the `error` handler of the `$logger` property in the `Vendor\ModuleName\Observer\MyObserver`.
+
+   ```xml
+   ...
+   <argument name="handlers" xsi:type="array">
+       <item name="error" xsi:type="object">Vendor\ModuleName\Logger\Handler\ErrorHandler</item>
+   </argument>
+   ...
+   ```
+
+   It will log exception messages into `/var/log/my_custom_logger/error.log` file.
```

Comment:
No indentation. Rewrite the sentence to be similar to the one at the end of the previous section.